tests/test_pulse.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from libmozevent.pulse import PulseListener
+
+
+def test_matching_routes():
```

Comment:
Could you add the real cases from https://github.com/mozilla/code-review/blob/d510e77eedfc329f471938d559026af9d08243c8/events/tests/test_bugbug_utils.py#L174 and https://github.com/mozilla/code-review/blob/d510e77eedfc329f471938d559026af9d08243c8/events/tests/test_bugbug_utils.py#L371 as test cases?

Reply:
Done in 033f77690bd6d10aa02fd0050f0fb948c76214d6